app/src/pages/inside/projectSettingsPageContainer/content/analyzerContainer/analyzerContainer.jsx

```diff
@@ -38,6 +38,8 @@ import { analyzerExtensionsSelector } from 'controllers/appInfo';
 import { canUpdateSettings } from 'common/utils/permissions';
 import { activeProjectRoleSelector, userAccountRoleSelector } from 'controllers/user';
 import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
+import { projectOrganizationSlugSelector } from 'controllers/project/selectors';
```

Comment:
Why not to extract it from URL?

Reply:
I thin that we not always have organizationSlug property in our URL. Because backend dont pass it in assignedProjects and projects respones